### DISCUSSION TITLE
Smartwings WM25L-Z Showing as mains powered since recharging via powerbank

### DISCUSSION THREAD

**Discussion by mikehedger** [Koenkk/zigbee2mqtt#21683] [Devices]
Since charging all 5 of my rollerblinds they show in Z2M as a mains powered device and no longer offers a battery percentage. This has happened even after a repair. Does anyone have any suggestions to get this changed back to a battery powered device?

![Screenshot_20240305_115339](https://github.com/Koenkk/zigbee2mqtt/assets/136187016/c51f8618-f2ff-4476-a902-33d997fe74ac)

**Comment by pfremm** [comment]
Did you ever figure this out @mikehedger?  Mine also show mains.

**Reply by mikehedger** [reply]
I forgot all about this, I've now paired them to my IKEA Dirigera hub.
